webapp/src/routes/OffersManagement/OffersManagement.js

```diff
@@ -165,26 +162,39 @@ const OffersManagement = () => {
         break
     }
   }
+
+  const [anchorEl, setAnchorEl] = useState(null)
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget)
+  };
+
+  const handleClose1 = () => {
+    setAnchorEl(null)
+  };
+
   const Actions = (active, offer_id) => (
-    <FormControl variant="filled" className={classes.formControl}>
-      <InputLabel id="actions-selection-label">
-        {t('offersManagement.action')}
-      </InputLabel>
-      <Select
+    <div>
+      <Button aria-controls="simple-menu" aria-haspopup="true" onClick={handleClick}>
+        <MoreVertIcon />
+      </Button>
+      <Menu
+        id="simple-menu"
+        anchorEl={anchorEl}
+        keepMounted
         value=""
-        onClick={(e) => handleActionClick(e.target.value, active, offer_id)}
-        labelId="actions-selection"
-        id="action-select"
+        open={Boolean(anchorEl)}
+        onClose={handleClose1}
       >
-        <MenuItem value="edit">{t('common.edit')}</MenuItem>
-        <MenuItem value="delete">{t('common.delete')}</MenuItem>
-        <MenuItem value={active ? 'deactivate' : 'activate'}>
+        <MenuItem onClick={(e) => handleActionClick("edit", active, offer_id)}>{t('common.edit')}</MenuItem>
+        <MenuItem onClick={(e) => handleActionClick("delete", active, offer_id)}>{t('common.delete')}</MenuItem>
+        <MenuItem onClick={(e) => handleActionClick(active ? 'deactivate' : 'activate', active, offer_id)}>
```

Comment:
@steph222 is **_deactivate_** word right?

Reply:
It is the word that was implemented from the beginning  
![image](https://user-images.githubusercontent.com/31549144/120284767-d5e02500-c279-11eb-8f12-f5854467f31d.png)